src/chai/ManagedArray.hpp

```diff
@@ -449,9 +450,10 @@ class ManagedArray : public CHAICopyable
   /*!
    * Pointer to PointerRecord data.
    */
-  mutable PointerRecord* m_pointer_record = nullptr;
 
   mutable bool m_is_slice = false;
+public:
+  mutable PointerRecord* m_pointer_record = nullptr;
```

Comment:
Why is this now public?

Reply:
This is left over from debugging that weird behaviour, I'll fix it.